userguide/tutorials/payment_plugin.adoc

```diff
@@ -1,44 +1,37 @@
 = Payment Plugin Overview
 
-Payment plugins integrate Kill Bill with a specific gateway (i.e. payment processor), such as Stripe or Braintree. We already have many open-source payment plugins available on GitHub as follows:
+Payment plugins integrate Kill Bill with a specific https://docs.killbill.io/latest/Kill-Bill-Glossary.html#payment_gateway[gateway] (i.e. payment processor), such as Stripe or Braintree. We already have many open-source payment plugins available on GitHub as listed in our https://docs.killbill.io/latest/plugin_introduction.html#payment_gateways[_plugin document_].
 
-. https://github.com/killbill/killbill-stripe-plugin[_killbill-stripe-plugin_] - Plugin to use  https://stripe.com/[_Stripe_] as the payment gateway
+This tutorial can be used to develop a custom payment plugin.
 
-. https://github.com/killbill/killbill-adyen-plugin[_killbill-adyen-plugin_] - Plugin to use https://www.adyen.com/[_Adyen_] as the payment gateway
+== Prerequisites
 
-. https://github.com/killbill/killbill-qualpay-plugin[_killbill-qualpay-plugin_] - Plugin to use https://www.qualpay.com/[_QualPay_] as the payment gateway  
+* Make sure to familiarize yourself with https://docs.killbill.io/latest/userguide_subscription.html#components-payment[_payment components_] and the https://docs.killbill.io/latest/userguide_payment.html[_payment userguide_].
 
-This guide will give you pointers in case you need to develop your own payment plugin.
+* Make sure to familiarize yourself with https://docs.killbill.io/latest/plugin_introduction.html[_Kill Bill plugins_].
 
-Before reading this guide, make sure to familiarize yourself with https://docs.killbill.io/latest/userguide_subscription.html#components-payment[payment components] and the https://docs.killbill.io/latest/userguide_payment.html[payment userguide].
 
 == Developing a Payment Plugin
 
-We provide an interface called https://github.com/killbill/killbill-plugin-api/blob/master/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java[PaymentPluginApi]. This is the interface between Kill Bill and payment plugins. In order to create your own payment plugin, you need to create a class that implements this interface and implement the methods in this interface.
+Kill Bill provides a  https://github.com/killbill/killbill-plugin-api/[_Plugin API_] for plugin development. This includes the https://github.com/killbill/killbill-plugin-api/blob/master/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java[_PaymentPluginApi_] interface which can be used to develop a payment plugin. Thus, this interface needs to be implemented in order to develop a custom payment plugin.
```

Comment:
```suggestion
Kill Bill provides a https://github.com/killbill/killbill-plugin-api/[Plugin API] for plugin development. This includes the https://github.com/killbill/killbill-plugin-api/blob/master/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java[`PaymentPluginApi`] interface, which you can use to develop a payment plugin. Thus, you need to implement this interface in order to develop a custom payment plugin.
```

Reply:
> This includes the `PaymentPluginApi` interface, which you can use to develop a payment plugin. Thus, you need to implement this interface in order to develop a custom payment plugin.

It sounds like we're saying the same thing twice: "...interface, which you can use to develop a payment plugin" and "you need to implement this interface in order to develop a customer payment plugin."

Reply:
Fixed